lib/Munin/Node/Configure/Plugin.pm

```diff
@@ -254,10 +254,11 @@ sub parse_suggest_response
         }
     }
 
-    unless (@{ $self->{suggestions} }) {
-        $self->log_error("No valid suggestions");
-        return;
-    }
+    # ignore empty suggest response
```

Comment:
Please do not leave dead code in comments around :)

Is this change necessary combined with the change below?
Maybe add some flag, whether an empty response should be ignored? (just guessing - I did not take a closer look)

Reply:
> dead code in comments

That's indeed always bad in commited code.